templates/field/field--media--remote-video.html.twig

```diff
@@ -38,6 +38,7 @@
 #}
 {% for item in items %}
   {% include '@components/video/video.twig' with {
+    'attributes': attributes,
```

Comment:
Since we're looping through the field items at this point, should the attributes be `item.attributes` instead? Same question applies to all templates where we're trying to pass attributes into a template to render a field item.